cylc/flow/scheduler.py

```diff
@@ -1470,7 +1470,7 @@ async def main_loop(self):
 
             self.process_command_queue()
 
-            if self.pool.release_runahead_tasks():
+            if not self.is_paused and self.pool.release_runahead_tasks():
```

Comment:
Hmm, this would appear to be the reverse of a previous commit:

https://github.com/cylc/cylc-flow/commit/22e7a202ff5ad3c8e6becdc66beeea2df2e3f3e3#diff-90711b005fe27b9f894e6cd09f08816d847dcf853518894c6b7e2139784cbcb8R1444

Reply:
I think runahead tasks should still be released in "paused" mode, but that the "paused" state should stop them from running.

https://github.com/cylc/cylc-flow/pull/4436

Reply:
Good spotting! Admittedly this atempted fix was done in a desperate rush.